config.m4

```diff
@@ -19,15 +21,29 @@ if test "$PHP_UUID" != "no"; then
   PHP_CHECK_FUNC_LIB(uuid_generate_md5, uuid)
   PHP_CHECK_FUNC_LIB(uuid_generate_sha1, uuid)
 
+  PHP_CHECK_FUNC_LIB(uuid_time, uuid)
+  PHP_CHECK_FUNC_LIB(uuid_mac, uuid)
+
   export OLD_CPPFLAGS="$CPPFLAGS"
   export CPPFLAGS="$CPPFLAGS $INCLUDES -DHAVE_UUID"
   AC_CHECK_HEADER([uuid/uuid.h], [], AC_MSG_ERROR('uuid/uuid.h' header not found))
-  PHP_SUBST(UUID_SHARED_LIBADD)
 
-  PHP_ADD_LIBRARY_WITH_PATH(uuid, $PHP_UUID_DIR/$PHP_LIBDIR, UUID_SHARED_LIBADD)
+  AC_MSG_CHECKING(PHP version)
+  AC_TRY_COMPILE([], [
+#ifdef __APPLE__
+#error compiling on darwin
+#endif
+],
+[],
+[export UUID_ON_MAC=true])
+
+
+  if test $UUID_ON_MAC != "true"; then
+    PHP_SUBST(UUID_SHARED_LIBADD)
+    PHP_ADD_LIBRARY_WITH_PATH(uuid, $PHP_UUID_DIR/lib, UUID_SHARED_LIBADD)
```

Comment:
Please use `$PHP_LIBDIR`

Reply:
Like this?
```suggestion
    PHP_ADD_LIBRARY_WITH_PATH(uuid, $PHP_LIBDIR, UUID_SHARED_LIBADD)
```

Reply:
See deleted line above... `$PHP_UUID_DIR/$PHP_LIBDIR`